### PR TITLE
Add 2020 RC

### PR DESCRIPTION
**Pull request by paragpachpute**

### CODE REVIEW

File: venues/ML_Reproducibility_Challenge/2020/process/commentProcess.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def process(client, note, invitation):
+    SHORT_PHRASE = "NeurIPS Reproducibility Challenge"
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/process/commentProcess.py:2]:
```suggestion
    SHORT_PHRASE = "ML Reproducibility Challenge 2020"
```

File: venues/ML_Reproducibility_Challenge/2020/python/create_neurips_papers.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import argparse
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/python/create_neurips_papers.py:1]: I don't think you need this script.

Comment (melisabok) [reply venues/ML_Reproducibility_Challenge/2020/python/create_neurips_papers.py:1]: you can remove it

File: venues/ML_Reproducibility_Challenge/2020/python/post_claims.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    signatures=['~Super_User1'],
+    members=[],
+    details={ 'writable': True })
+)
+
+claim_inv = client.post_invitation(openreview.Invitation(
+    id='{}/-/Claim'.format(conference_id),
+    readers=['everyone'],
+    invitees=['~'],
+    writers=[conference_id],
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/python/post_claims.py:29]: this invitation is missing the 'noninvitees' field, this is being used to allow the users to post only one claim at a time. But I think that using 'multiReply': false should work.

File: venues/ML_Reproducibility_Challenge/2020/python/setup.py [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+client = openreview.Client(baseurl=baseUrl, username=superUser, password=password)
+
+
+CONFERENCE_ID = 'ML_Reproducibility_Challenge/2020'
+PROGRAM_CHAIRS_ID = CONFERENCE_ID + '/Program_Chairs'
+SUBMISSION_ID = CONFERENCE_ID + '/-/Submission'
+AUTHORS_ID = CONFERENCE_ID + '/Authors'
+ACCEPTED_PAPER_ID = CONFERENCE_ID + '/-/Accepted_Papers'
+
+# builder = openreview.builder.ConferenceBuilder(client)
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/python/setup.py:29]: you can remove the commented code

File: venues/ML_Reproducibility_Challenge/2020/python/setup.py [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+# client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+baseUrl = "http://localhost:3000"
+superUser = 'OpenReview.net'
+password = 'OpenReview_dev'
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/python/setup.py:19]: can you remove this from the PR?

File: venues/ML_Reproducibility_Challenge/2020/process/commentProcess.py (hunk 1 above)
Comment (paragpachpute) [reply venues/ML_Reproducibility_Challenge/2020/process/commentProcess.py:2]: done

File: venues/ML_Reproducibility_Challenge/2020/python/create_neurips_papers.py (hunk 2 above)
Comment (paragpachpute) [reply venues/ML_Reproducibility_Challenge/2020/python/create_neurips_papers.py:1]: done

File: venues/ML_Reproducibility_Challenge/2020/python/post_claims.py (hunk 3 above)
Comment (paragpachpute) [reply venues/ML_Reproducibility_Challenge/2020/python/post_claims.py:29]: I am not sure if this file is even being used, seems like it is an ad-hoc file

File: venues/ML_Reproducibility_Challenge/2020/webfield/landingWebfield.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+var GROUP_ID = '';
```
Comment (melisabok) [inline venues/ML_Reproducibility_Challenge/2020/webfield/landingWebfield.js:1]: you can remove this file

File: venues/ML_Reproducibility_Challenge/2020/python/load_papers.py [hunk 7]
```diff
@@ -38,7 +38,7 @@
 
 def post_paper(paper, client):
     authors = ""
-    if not math.isnan(paper['Authors']):
+    if paper['Authors']:
```
Comment (paragpachpute) [inline venues/ML_Reproducibility_Challenge/2020/python/load_papers.py:41]: I think the papers for which we got the error were venue names. Do we want to add the venues as accepted papers?

**Review by melisabok — APPROVED** [review]

**Pull request merged**